### DISCUSSION TITLE
v2.7.0

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4095] [分享 / Share]
✨新增
- 快捷键扩展的几个插件自带了 YouTube / PotPlayer 等预设的默认按键. (#3971)
- `简化直播间` 在开启隐藏付费礼物时, 将直接隐藏左下角的打榜入口 (以前是领银瓜子所以没算进付费礼物). (#4067)
- `专栏文字选择` 更名为 `专栏复制优化`, 现在专栏的文字默认就可以直接选择, 本功能可以用于去除复制后带上的多余文本. (#4065)
- 删除功能 `直播录像下载`, 现在已经没有直播录像的页面了. (#4061)
- 在检测到[解除番剧区域限制](https://greasyfork.org/zh-CN/scripts/25718-%E8%A7%A3%E9%99%A4b%E7%AB%99%E5%8C%BA%E5%9F%9F%E9%99%90%E5%88%B6)的脚本在运行时, 本脚本将停止运行以避免兼容性问题. (#2704)
- `选集区域优化` 在多个分组时支持仅展开当前分组. (#3899)
- `播放时自动关灯` 支持星光动画. (PR #4077 by [z503722728](https://github.com/z503722728))

🐛修复
- 修复 `直播信息扩充` 失效的问题. (#4034, PR #4043 by [deepdarkssj](https://github.com/deepdarkssj))
- 修复 `简化直播间` 在开启隐藏付费礼物时, 右下布局错乱. (PR #4089 by [Pencil](https://github.com/pencilqaq))
- 修复动态相关功能对新版专栏卡片不生效. (#3994)
- 修复搜索界面的广告链接仍然可点击. (#3969)
- 修复 `图片批量导出` 在动态失效了. (#4038)
- 修复 `选集区域优化` 仅开启 `展开选集标题` 时视频标题无法显示完全. (#3909)
- 修复 `查看封面` 在复制链接后没有显示成功图标. (#4002)
- 修复番剧页面的部分夜间模式. (PR #4077 by [z503722728](https://github.com/z503722728))

☕开发者相关
- 修复 `IframePopup.vue` 的类型定义. (PR #4014 by [timongh](https://github.com/timongh))
- 修复 `UserItem.vue` 初始化 Toast 的逻辑问题. (PR #4056 by [timongh](https://github.com/timongh))
- 优化了 `switch-options.ts` 的注释. (PR #4021 by [timongh](https://github.com/timongh))
- 清理了项目中 Vue 文件里对未定义字段的引用. (#3992)
- 导出了 Dialog API (`@/core/dialog`), 允许其他组件使用.
- 调整了 CONTRIBUTING 中的本地开发流程描述. (PR #4077 by [z503722728](https://github.com/z503722728))
- 增加了 GitHub Projects 看板: https://github.com/users/the1812/projects/1/views/3, 上面会列出当前和未来计划的功能, 可供参考.
![image](https://user-images.githubusercontent.com/26504152/227761813-fe2a1c76-184e-4d23-a75a-7b43871b5e05.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.7.0'>v2.7.0</a>.</em>

**Comment by SilentEagleMSZ** [comment]
更新之后脚本失效 退回2.6.3版本就好了

**Comment by the1812** [comment]
> 在检测到[解除番剧区域限制](https://greasyfork.org/zh-CN/scripts/25718-%E8%A7%A3%E9%99%A4b%E7%AB%99%E5%8C%BA%E5%9F%9F%E9%99%90%E5%88%B6)的脚本在运行时, 本脚本将停止运行以避免兼容性问题.

关于这一条功能, 目前强制开启仍然不是很好的办法 (#4100), 我加了一个 `兼容性选项` 的设置允许自定义这个行为, 重新下载最新的脚本覆盖安装即可.

对应的版本号如下:
- 正式版 (GitHub 源): `v2.7.0-42-gc2ba4a36a`
- 正式版 (jsDelivr 源): `v2.7.0-46-g91e80d9c5`
- 预览版: `v2.7.0-preview-70-gabd8040b8`

<img width="400" src="https://user-images.githubusercontent.com/26504152/227785768-e1e6c5cd-4417-4500-b4e6-0202e36a3ca3.png">

**Reply by AzusaHana** [reply]
“解除b站区域限制”重构的页面仅是 https://www.bilibili.com/bangumi/play/
在其他诸如 https://www.bilibili.com/bangumi/media/ 等页面不会触发重构，Bilibili Evolved 能正常使用
是否能多加一个判断条件

**Reply by the1812** [reply]
下个版本会只限制在 bangumi 页面触发

**Comment by Phylirui** [comment]


**Reply by Phylirui** [reply]
B站返回旧版就可以了，脚本不支持新版b站。

**Reply by Geeyun-JY3** [reply]
这问题真是每个版本各种浏览器都有人报告，不完全统计目前 issue 还开着的就还有：#2927、#3747、#3918、#3988、#4030、#4121

**Comment by Shaw5A** [comment]
在使用简化直播间屏蔽付费礼物后，网页全屏和全屏时下方礼物栏出现灰色区域，使画面不能真正全屏
![1](https://user-images.githubusercontent.com/88639398/230695203-c639955c-e58a-4fdf-afb8-86f804d74cf9.PNG)

**Reply by AzusaHana** [reply]
那里有个🎁符号，可以关闭礼物栏

**Reply by Geeyun-JY3** [reply]
善用搜索：#4108

**Reply by Shaw5A** [reply]
好了，谢谢

**Comment by HAOZ666** [comment]
![image](https://user-images.githubusercontent.com/68998938/230721525-3b3fe0ea-aec7-4c0b-a09e-5083801c759a.png)
请问，夜间模式下 视频背景的白边怎么去除？

**Reply by Geeyun-JY3** [reply]
建议附带更多信息，不然很难帮你确定问题。

**Reply by HAOZ666** [reply]
![image](https://user-images.githubusercontent.com/68998938/230730671-4b66a0a1-5b4c-4e06-89c1-9e03f3ffd347.png)
![image](https://user-images.githubusercontent.com/68998938/230730838-ac8393f0-23b3-4cb7-bf9f-4366fef41988.png)

**Reply by HAOZ666** [reply]
> 建议附带更多信息，不然很难帮你确定问题。

你好 夜间模式弹幕栏这里应该是保持一致的，白天模式都是白色 ，夜间模式都是暗色 ，

**Reply by Geeyun-JY3** [reply]
我是指播放器版本、脚本版本等信息。

**Reply by HAOZ666** [reply]
![image](https://user-images.githubusercontent.com/68998938/230751516-d8f85729-3c6d-43a8-b051-16fec7512bcd.png)
![image](https://user-images.githubusercontent.com/68998938/230751812-bd3303cc-0698-47f3-a875-ef19eec145a9.png)

**Reply by Geeyun-JY3** [reply]
建议你先更新全部组件到最新版，我这边新版播放页面一样的播放器版本一样的脚本版本无法复现。

**Reply by HAOZ666** [reply]
好 thks
